diff -uaNr nginx-http-flv-module/ngx_http_flv_live_module.c nginx-http-flv-module-bugfix/ngx_http_flv_live_module.c
--- nginx-http-flv-module/ngx_http_flv_live_module.c	2020-07-31 15:56:27.318120854 +0800
+++ nginx-http-flv-module-bugfix/ngx_http_flv_live_module.c	2020-07-31 15:54:11.142117368 +0800
@@ -14,6 +14,10 @@
 static ngx_rtmp_close_stream_pt next_close_stream;
 
 
+ngx_rtmp_play_pt                http_flv_live_next_play;
+ngx_rtmp_close_stream_pt        http_flv_live_next_close_stream;
+
+
 static ngx_int_t ngx_http_flv_live_init(ngx_conf_t *cf);
 static void *ngx_http_flv_live_create_loc_conf(ngx_conf_t *cf);
 static char *ngx_http_flv_live_merge_loc_conf(ngx_conf_t *cf,
diff -uaNr nginx-http-flv-module/ngx_http_flv_live_module.h nginx-http-flv-module-bugfix/ngx_http_flv_live_module.h
--- nginx-http-flv-module/ngx_http_flv_live_module.h	2020-07-31 15:56:28.846120893 +0800
+++ nginx-http-flv-module-bugfix/ngx_http_flv_live_module.h	2020-07-31 15:54:11.142117368 +0800
@@ -21,8 +21,8 @@
 extern ngx_module_t ngx_rtmp_module;
 
 
-ngx_rtmp_play_pt         http_flv_live_next_play;
-ngx_rtmp_close_stream_pt http_flv_live_next_close_stream;
+extern ngx_rtmp_play_pt                http_flv_live_next_play;
+extern ngx_rtmp_close_stream_pt        http_flv_live_next_close_stream;
 
 
 #define ngx_rtmp_cycle_get_module_main_conf(cycle, module)                \

